src/test/java/com/google/api/codegen/testdata/nodejs_main_library.baseline

```diff
@@ -102,6 +88,8 @@ var ALL_SCOPES = [
  *   Custom function to obtain the credentials.
  * @param {grpc.ClientCredentials} opts.sslCreds
  *   A ClientCredentials for use with an SSL-enabled channel.
+ * @param {Object} opts.grpc
```

Comment:
So, once this is added to gcloud-node, if I'm a gcloud-node user, I have to set this parameter in order to make the library work?